src/systems/systems/src/navigation/adirs.rs

```diff
@@ -1031,6 +1053,8 @@ impl InertialReference {
             SignStatus::NoComputedData
         };
 
+        self.true_heading
+            .set_value(simulator_data.true_heading, ssm);
```

Comment:
The heading should become available prior to full allignment, whereas the ssm you're using here does not.